### PR TITLE
Revert "Temporarily disable fuzzing"

### PR DESCRIPTION
**Pull request by apoelstra**
This reverts commit 720ea29865c64e03a2b0c872e20e0dbf99cf305e.

There is a new version of honggfuzz out. Maybe things work now.

### CODE REVIEW

**Comment by apoelstra** [comment]
:/ now it seems like fuzzing is not being done by CI. I vote we just merge this and see what happens.

**Comment by apoelstra** [comment]
Ah, nice, thanks @TheBlueMatt! I'd forgotten about this. DIsabling features for honggfuzz is much cleaner than pinning `arbitrary`.

**Comment by apoelstra** [comment]
Well, these errors are incomprehensible :/. And I can't reproduce them locally.

**Comment by apoelstra** [comment]
Ok I can reproduce. It's a bash quoting ithngi.

**Comment by apoelstra** [comment]
Ok, the current error is a "how did this ever work" error, so we're making progress :)

**Comment by apoelstra** [comment]
CI is passing but it didn't run the fuzz units and I don't know why :(

**Comment by apoelstra** [comment]
Huzzah!!

File: fuzz/travis-fuzz.sh [hunk 1]
```diff
@@ -8,9 +8,17 @@ if [ ${incorrectFilenames} -gt 0 ]; then
...
+else
+	TARGETS=fuzz_targets/"$1".rs
+fi
+
 # Testing
 cargo install --force honggfuzz --no-default-features
-for TARGET in fuzz_targets/*; do
+for TARGET in $TARGETS; do
+	echo $TARGET
+	continue
```
Comment (sanket1729) [inline fuzz/travis-fuzz.sh:21]: Is this even doing the fuzzing? Won't the `continue` just skip rest of the loop

Comment (apoelstra) [reply fuzz/travis-fuzz.sh:21]: lol oops this was test code

**Review by sanket1729 — APPROVED** [review]
utACK d25aba8ca0225a560e88e183b0db69dd1bd019a1

**Review by tcharding — APPROVED** [review]
ACK d25aba8ca0225a560e88e183b0db69dd1bd019a1

**Comment by tcharding** [comment]
Cheers bro.

**Pull request merged**